modules/oe_content_tender/config/optional/language.content_settings.node.oe_tender.yml

```
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.oe_tender
```

you are missing something like
```suggestion
    - node.type.oe_tender
module:
    - content_translation
third_party_settings:
  content_translation:
    enabled: false
```

---

Doing `drush cde oe_content_tender -y` it is not added so I don't think if it is necessary.
Am I wrong ?

---

I think it is needed, I think that this is a manual change in preparation of ewcms integration, that is probably why it is not added to config devel export

---

Fixed